whois-api/src/test/java/net/ripe/db/whois/api/rdap/WhoisRdapServiceTestIntegration.java

```diff
@@ -2283,6 +2291,74 @@ public void lookup_org_entity() {
+                "language:      EN\n" +
+                "admin-c:       TP2-TEST\n" + //has email and notify
+                "tech-c:        TP1-TEST\n" +
+                "tech-c:        TP2-TEST\n" + //has email and notify
+                "mnt-ref:       OWNER-MNT\n" +
+                "mnt-ref:       OWNER-MNT\n" +
+                "mbrs-by-ref:   OWNER-MNT, ANOTHER-MNT\n" +
+                "mnt-by:        ANOTHER-MNT\n" +
+                "created:         2011-07-28T00:35:42Z\n" +
+                "last-modified:   2019-02-28T10:14:46Z\n" +
+                "source:        TEST");
+
+        final Entity entity = createResource("entity/ORG-ONE-TEST")
+                .request(MediaType.APPLICATION_JSON_TYPE)
+                .get(Entity.class);
+
+        assertThat(entity.getRedacted().size(), is(4));
+
+        assertThat(entity.getRedacted().get(0).getName().getDescription(), is("Updates notification e-mail information"));
+        assertThat(entity.getRedacted().get(0).getReason().getDescription(), is("Personal data"));
+        assertDoesNotThrow(() -> JsonPath.compile(entity.getRedacted().get(0).getPrePath()));
+        assertThat("$.entities[?(@.roles=='registrant')].vcardArray[1][?(@[0]=='notify')]",
+                is(entity.getRedacted().get(0).getPrePath()));
+        assertThat(entity.getRedacted().get(0).getMethod(), is("removal"));
+
+        assertThat(entity.getRedacted().get(1).getName().getDescription(), is("Updates notification e-mail information"));
+        assertThat(entity.getRedacted().get(1).getReason().getDescription(), is("Personal data"));
+        assertDoesNotThrow(() -> JsonPath.compile(entity.getRedacted().get(1).getPrePath()));
+        assertThat("$.entities[?(@.roles=='technical && administrative')].vcardArray[1][?(@[0]=='notify')]", is(entity.getRedacted().get(1).getPrePath()));
+        assertThat(entity.getRedacted().get(1).getMethod(), is("removal"));
+
+        assertThat(entity.getRedacted().get(2).getName().getDescription(), is("Authenticate incoming references"));
+        assertThat(entity.getRedacted().get(2).getReason().getDescription(), is("No registrant mntner"));
+        assertDoesNotThrow(() -> JsonPath.compile(entity.getRedacted().get(2).getPrePath()));
+        assertThat("$.entities[?(@.handle=='OWNER-MNT')]", is(entity.getRedacted().get(2).getPrePath()));
+        assertThat(entity.getRedacted().get(2).getMethod(), is("removal"));
+
+        assertThat(entity.getRedacted().get(3).getName().getDescription(), is("Authenticate members by reference"));
+        assertThat(entity.getRedacted().get(3).getReason().getDescription(), is("No registrant mntner"));
+        assertDoesNotThrow(() -> JsonPath.compile(entity.getRedacted().get(3).getPrePath()));
+        assertThat("$.entities[?(@.handle=='OWNER-MNT,ANOTHER-MNT')]", is(entity.getRedacted().get(3).getPrePath()));
+        assertThat(entity.getRedacted().get(3).getMethod(), is("removal"));
```

Comment:
Correct me if I am wrong, we have individual keys we never send comma separated multiple values in one handle in RDAP?

Reply:
let me check this

Reply:
yes, you are right it should come in two different redactions

Reply:
Done